Src/WitsmlExplorer.Frontend/components/Modals/CopyRangeModal.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { useContext, useState } from "react";
+import NavigationContext from "../../contexts/navigationContext";
+import OperationContext from "../../contexts/operationContext";
+import OperationType from "../../contexts/operationType";
+import { ComponentType } from "../../models/componentType";
+import { CopyRangeClipboard, createComponentReferences } from "../../models/jobs/componentReferences";
+import { indexToNumber } from "../../models/logObject";
+import { WITSML_INDEX_TYPE_DATE_TIME, WITSML_INDEX_TYPE_MD } from "../Constants";
+import ModalDialog from "./ModalDialog";
+import AdjustDateTimeModal from "./TrimLogObject/AdjustDateTimeModal";
+import AdjustNumberRangeModal from "./TrimLogObject/AdjustNumberRangeModal";
+
+export interface CopyRangeModalProps {
+  mnemonics: string[];
+}
+
+const CopyRangeModal = (props: CopyRangeModalProps): React.ReactElement => {
+  const {
+    navigationState: { selectedServer, selectedLog }
+  } = useContext(NavigationContext);
+  const { dispatchOperation } = useContext(OperationContext);
+  const [startIndex, setStartIndex] = useState<string | number>();
+  const [endIndex, setEndIndex] = useState<string | number>();
+  const [confirmDisabled, setConfirmDisabled] = useState<boolean>();
```

Comment:
Initialize with true so the Copy button starts disabled.

```suggestion
  const [confirmDisabled, setConfirmDisabled] = useState<boolean>(true);
```